### PR TITLE
Attempting forge recipe for tyssue

### PR DESCRIPTION
**Pull request by glyg**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -12,10 +12,6 @@ environment:
     # Note: Because we have to separate the py2 and py3 components due to compiler version, we have a race condition for non-python packages.
```
Comment (synapticarbors) [inline appveyor.yml]: Please do not edit this file.

File: recipes/tyssue/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  license: MPLv2.0
+  summary: 'A tissue simulation library'
+  description: |
+    tyssue provides data structures and algorihms to model epithelium
+    in 2D and 3D for python 3.4+
+  doc_url: http://tyssue.readthedocs.io
+  dev_url: https://github.com/DamCB/tyssue
+extra:
+  recipe-maintainers:
+    - Guillaume Gay / glyg
```
Comment (synapticarbors) [inline recipes/tyssue/meta.yaml]: Please just list your github username only.

File: recipes/tyssue/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  sha256: {{ sha256 }}
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```
Comment (synapticarbors) [inline recipes/tyssue/meta.yaml]: Are numpy and scipy required to build the recipe? If not, please delete these lines.

File: recipes/tyssue/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    - scipy
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - pandas
+    - pytables
+    - jupyter
+    - notebook
```
Comment (synapticarbors) [inline recipes/tyssue/meta.yaml]: vispy is listed a requirement in: 

https://github.com/DamCB/tyssue/blob/master/doc/requirements.txt

File: recipes/tyssue/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+test:
+  imports:
+    - tyssue
```
Comment (synapticarbors) [inline recipes/tyssue/meta.yaml]: Please add an import for each submodule, e.g. `- tyssue.behaviors`.

File: recipes/tyssue/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+about:
+  home: https://github.com/DamCB/tyssue
+  license: MPLv2.0
+  summary: 'A tissue simulation library'
+  description: |
+    tyssue provides data structures and algorihms to model epithelium
+    in 2D and 3D for python 3.4+
```
Comment (synapticarbors) [inline recipes/tyssue/meta.yaml]: If this package is really only python 3.4+, please use the appropriate "skip" in the build:

http://conda.pydata.org/docs/building/meta-yaml.html#skipping-builds

File: recipes/tyssue/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    - tyssue.io
+    - tyssue.particles
+    - tyssue.solvers
+    - tyssue.stores
+    - tyssue.topology
+    - tyssue.utils
+
+about:
+  home: https://github.com/DamCB/tyssue
+  license: MPLv2.0
```
Comment (glyg) [inline recipes/tyssue/meta.yaml]: Sure, done!

Thanks @synapticarbors and @scopatz for the review!!

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**